packages/arb-avm-cpp/data_storage/src/value/machine.cpp

```diff
@@ -217,18 +244,21 @@ rocksdb::Status saveMachineState(Transaction& transaction,
     machine_state_keys.arb_gas_remaining = machinestate.arb_gas_remaining;
     machine_state_keys.pc = machinestate.pc;
     machine_state_keys.err_pc = machinestate.errpc;
-    machine_state_keys.staged_message_hash =
-        hash_value(machinestate.staged_message);
-    machine_state_keys.total_messages_consumed =
-        machinestate.total_messages_consumed;
+    machine_state_keys.messages_fully_processed = machinestate.messages_fully_processed;
+    machine_state_keys.inbox_accumulator = machinestate.inbox_accumulator;
+    machine_state_keys.staged_message = machinestate.staged_message_temp;
     machine_state_keys.status = machinestate.state;
 
     return rocksdb::Status::OK();
 }
 
 SaveResults saveMachine(Transaction& transaction, const Machine& machine) {
```

Comment:
Are we actually using this anymore? If not we could probably remove it at this pint

Reply:
still using, mostly in unit tests. future refactor